### ISSUE TITLE
When `assemble-windows` CI job failed, the corresponding GCP instance should not be left dangling

### ISSUE THREAD

**Issue by lolski** [graknlabs/workbase#157]
Reproducing the issue:
- This issue will show up when the job failed for any reason.

Example:
- the following job failed and left a dangling GCP instance that had to be killed manually: https://circleci.com/gh/Irtazaraza/workbase/408

**Comment by vmax** [comment]
No longer needed after #229
